conda/cli/activate.py

```diff
@@ -32,11 +35,27 @@ def prefix_from_arg(arg):
 
 
 def binpath_from_arg(arg):
-    path = join(prefix_from_arg(arg), 'bin')
-    if not isdir(path):
-        sys.exit("Error: no such directory: %s" % path)
+    if sys.platform == "win32":
+        path = [prefix_from_arg(arg),
+                join(prefix_from_arg(arg), 'Scripts'),
+                join(prefix_from_arg(arg), 'Library/bin'),
```

Comment:
This is cleaner, definitely, but I think this would pass as harmless without the loop?  If dirs on PATH don't exist, then surely nothing is found in them?

Are there potential problems with having non-existent folders on PATH that I'm missing?